examples/app-demo/restaurant-backend/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use anyhow::Result;
+
+use slight_http_handler_macro::register_handler;
+
+wit_bindgen_rust::import!("../../../wit/http.wit");
+wit_error_rs::impl_error!(http::Error);
+
+wit_bindgen_rust::import!("../../../wit/mq.wit");
+wit_error_rs::impl_error!(mq::Error);
+
+fn main() -> Result<()> {
+    let router = http::Router::new()?;
+    let router_with_route = router
+        .post("/make_order", "handle_make_order")?
+        .get("/get_next_order", "handle_get_next_order")?;
```

Comment:
A `next` singleton resource hanging off of offers.
```suggestion
        .get("/orders/next", "handle_get_next_order")?;
```